packages/engine-server/src/drivers/sqlite/SqliteError.ts

```diff
@@ -0,0 +1 @@
+export class SqliteError extends Error {}
```

Comment:
fyi: played with a different way to manage error objects here https://github.com/dendronhq/dendron/pull/3744/commits/23ac7e5582134f3a63a70fb8db91eb8353bb0109.

Reply:
Yeah I want to include a better error typing, at the very least for sql based errors.  This is obviously just a stub right now that offers no functionality. Adding a TODO comment for now.